arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1958,55 +1962,25 @@
       "properties": {
         "provisioningState": {
           "type": "string",
+          "readOnly": true,
           "description": "Provisioning state of the namespace."
         },
-        "status": {
```

Comment:
@devigned, sorry for delay. We are reverting the properties related changes. We will be taking it in next API Version. we will revert and raise a new PR.